airbyte-integrations/bases/base-normalization/integration_tests/resources/test_simple_streams/data_input/catalog.json

```diff
@@ -159,7 +159,7 @@
         "default_cursor_field": []
       },
       "sync_mode": "incremental",
-      "cursor_field": ["_ab_cdc_lsn"],
+      "cursor_field": [],
```

Comment:
this replicates cdc behavior in practice, where source-postgres doesn't declare an explicit cursor field

(we should maybe reconsider that behavior, but I think that's a separate conversation)